install_config/install/advanced_install.adoc

```diff
@@ -264,20 +263,17 @@ master may require access to, or the installation will fail. Defaults to
 |This variable overrides the default subdomain to use for exposed
 xref:../../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[routes].
 
-|`openshift_master_image_policy_config`
-|Sets `imagePolicyConfig` in the master configuration. See xref:../../install_config/master_node_configuration.adoc#master-config-image-config[Image Configuration] for details.
-
 |`openshift_node_proxy_mode`
 |This variable specifies the
 xref:../../architecture/core_concepts/pods_and_services.adoc#service-proxy-mode[service
 proxy mode] to use: either `iptables` for the default, pure-`iptables`
 implementation, or `userspace` for the user space proxy.
 
-|`openshift_router_selector`
```

Comment:
Only applicable to 3.7 and 3.9
https://github.com/openshift/openshift-docs/pull/6609